pymonetdb/sql/pythonizebin.py

```diff
@@ -0,0 +1,378 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
you could also consider introducing a new namespace pythonise and split into bin and text submodules.